pkg/errors/errors.go

```diff
@@ -2,26 +2,8 @@ package errors
 
 import (
 	"fmt"
-
-	"github.com/pkg/errors"
 )
 
-func New(msg string) error {
```

Comment:
What about keeping these wrappers but just changing the implementation to use `fmt.Errorf`?

Reply:
These wrappers shadow the old functions that were deprecated and that we've been asked to move away from. I think the Go stdlib functions are more idiomatic and explicit...